src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -242,4 +290,45 @@ private String getUserId() {
     }
     return userId;
   }
+
+  // Gets formatted date for the string that is used in the email sent to users
+  private String getEmailDateString(TimeRange when) {
+    LocalDateTime start = LocalDateTime.ofInstant(when.start(), ZoneId.systemDefault());
+    String startTime = start.format(DateTimeFormatter.ofPattern("h:mm a"));
+    String day = start.format(DateTimeFormatter.ofPattern("EEEE, MMMM d, yyyy"));
+    return String.format("%s at %s UTC", day, startTime);
+  }
+
+  private String getEmail(String participantId) {
+    return personDao.get(participantId).map(Person::email).orElse("Nonexistent User");
+  }
+
+  private String getFirstName(String participantId) {
+    return personDao.get(participantId).map(Person::firstName).orElse("Nonexistent User");
+  }
+
+  private void sendParticipantEmail(String participantId, HashMap<String, String> emailedDetails)
+      throws IOException {
+    String subject = "You have been requested to conduct a mock interview!";
+    String contentString =
+        emailSender.fileContentToString(emailsPath + "/NewInterview_Interviewer.txt");
+    if (participantId.equals(getUserId())) {
+      subject = "You have been registered for a mock interview!";
+      contentString = emailSender.fileContentToString(emailsPath + "/NewInterview_Interviewee.txt");
+    }
+
+    Email recipient = new Email(getEmail(participantId));
```

Comment:
What happens when getEmail() returns "Nonexistant user"?
Should probably return

Reply:
Done